docs/faqs.json

```diff
@@ -1,3 +1,3 @@
 {
-  "text": "<h1 id=\"what-s-going-on-with-the-sis-view-\">What&#39;s going on with the SIS view?</h1>\n<p>It&#39;s … struggling. We are currently working on an update that should fix most of the problems.</p>\n<h1 id=\"other-question\">Other Question</h1>\n<p>I&#39;m another question.</p>\n"
+  "text": "<style>\nbody {\n  -webkit-overflow-scrolling: touch;\n  font-family: -apple-system, Roboto, sans-serif;\n  background-color: transparent;\n}\nh1 {\n  font-size: 1.35em;\n}\n</style>\n\n<h1 id=\"what-s-going-on-with-the-sis-view-\">What&#39;s going on with the SIS view?</h1>\n<p>It&#39;s … struggling. We are currently working on an update that should fix most of the problems.</p>\n<h1 id=\"other-question\">Other Question</h1>\n<p>I&#39;m another question.</p>\n"
```

Comment:
Halfway. We should never need to manually edit the file in `docs/`; instead, it's generated from the associated file(s) in `data/`.